cp/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/TaskService.java

```diff
@@ -52,6 +53,23 @@
 
   public List<TaskDto> readAll() {
     Bundle tasksBundle = taskRepository.findAllTasks();
+    FhirUtil.getFromBundle(tasksBundle, Task.class)
```

Comment:
Move this logic to converter. This is the whole idea of converters - you pass a bundle and it transforms it to dtos. Let it handle order/filler-order tasks itself.